command/operator_diagnose_test.go

```diff
@@ -66,17 +70,6 @@ func TestOperatorDiagnoseCommand_Run(t *testing.T) {
 				"-config", "./server/test-fixtures/nostore_config.hcl",
 			},
 			[]*diagnose.Result{
-				{
```

Comment:
Why are we removing these stanzas from the tests?

Reply:
#- [ ] []()

Reply:
In order not to have to update every test case when we add a new diagnose test, the stanzas are only including the results that are relevant to the test case itself.  This means we only have to add a new test case per test rather than updating all of the test case result trees.